control/control_standalone_test.go

```diff
@@ -24,7 +25,14 @@ import (
 	"golang.org/x/net/context"
 )
 
-func TestControl(t *testing.T) {
+func TestControlStandalone(t *testing.T) {
+	if os.Getuid() != 0 {
+		t.Skip("requires root")
+	}
+	if _, err := exec.LookPath("runc"); err != nil {
+		t.Skipf("no runc found: %s", err.Error())
+	}
```

Comment:
Probably these Skipfs can be moved to testutil pkg

Reply:
I didn't have a good way to combine with the `integration/sandbox` because I didn't want `New()` to have dependency on `testing`